libs/common-services/src/main/java/uk/gov/di/ipv/core/library/domain/cimitvc/ContraIndicator.java

```diff
@@ -1,11 +1,15 @@
 package uk.gov.di.ipv.core.library.domain.cimitvc;
 
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
+import lombok.NoArgsConstructor;
 import lombok.ToString;
 
 import java.util.List;
 
+@AllArgsConstructor
+@NoArgsConstructor(force = true)
```

Comment:
Why do we need the No-args constructor?

I wonder if we should just make this a `@Data`/`record` class instead

Reply:
@Joe-Edwards-GDS It seems the jackson library requires a no args contructor in order to deserialize the object - otherwise we get this exception
`Cannot construct instance of `uk.gov.di.ipv.core.library.domain.cimitvc.ContraIndicator` (no Creators, like default constructor, exist): cannot deserialize from Object value (no delegate- or property-based Creator)`